### ISSUE TITLE
chrome driver doesn't start on android simulator

### ISSUE THREAD

**Issue by tfikler** [appium/appium#19711]
### Do I have the most recent component updates?

- [X] I use the most recent available driver/plugin and server versions

### Is the component officially supported by the Appium team?

- [X] I have verified the component repository is present under the Appium organization in GitHub

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Until last week everything worked fine.
Last week, suddenly it stopped working.
When lunching android simulator, then open the chromedriver, it crashes.
Nothing changed in the code

Error:
Cannot synchronize local chromedrivers with the remote storage: unable to get local issuer certificate
Error: unable to get local issuer certificate

### Expected Behavior

I expect the chrome to open normally.

### Minimal Reproducible Example

from appium.options.android.uiautomator2.base import UiAutomator2Options
    desired_capabilities = {
        'platformName': 'Android',
        'platformVersion': '11.0',  # Update this to match the Android version of your emulator
        'deviceName': 'Pixel_7_API_30',  # Update this to match the AVD name of your emulator
        'showChromedriverLog': True,
        'chromedriver_autodownload': True,
        'browserName': 'chrome',
        'automationName': 'UiAutomator2',
        'noReset': True,
        'noSign': True,
        'avd': 'Pixel_7_API_30',
    }
    options = UiAutomator2Options()
    options.load_capabilities(desired_capabilities)
driver = webdriver.Remote(appium_url_server, options).

appium start command: appium --base-path /wd/hub --allow-insecure chromedriver_autodownload

    

### Environment

- Operating system: Mac M1
- Appium server version (output of `appium --version`): 2.4.1
- Appium driver(s) and their version(s): - uiautomator2@2.40.0 (automationName 'UiAutomator2')
- Node.js version (output of `node --version`):  v21.2.0
- `npm` version (output of `npm --version`): 6.14.18
- Platform and version under test: android emulator on android studio
- Real device or emulator/simulator: emulator

### Link to Appium Logs

https://gist.github.com/tfikler/b2594b7db09528a9988f385704dd7d87

**Comment by mykola-mokhnach** [comment]
This looks like an issue with your local configuration. The driver tries to connect to the Google storage to download the driver, but fails to do that because of cert validation issue. Try to download and provide the matching driver locally or provide a custom mirror as described in https://github.com/appium/appium-uiautomator2-driver#troubleshooting-chromedriver-download-issues

**Comment by mykola-mokhnach** [comment]
> still the same error, should i change another things in my configuration?

Unfortunately I cannot help there as I don't know how the root certificates are configured on your system.

> what does it means? download the chrome-driver locally and match it inside the desired capabilities as executable path?

yes, you need to download the correct chromedriver executable for your platform and for the target chrome version and then provide the path to it in capabilities.

**Comment by tfikler** [comment]
I managed to fix the problem.
I logged into the android device and check the chrome version installed (Probably it auto-updated)
I found the right chromedriver to download using this URL :
https://googlechromelabs.github.io/chrome-for-testing/known-good-versions-with-downloads.json
Paste only the chromedriver into the directory - `/Users/ec2-user/.appium/node_modules/appium-uiautomator2-driver/node_modules/appium-chromedriver/chromedriver/mac/chromedriver`
Add the capability - `chromedriverExecutable` with the path above.
and run appium without the autodownload.

Thanks for the help!
